Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:latest
+FROM debian:stretch
```

Comment:
`stretch` is quite old, and is already out of security support ([since 18 July 2020](https://en.wikipedia.org/wiki/Debian_version_history)). Did you try `buster`?